WordPress/WordPressTest/MediaRequestAuthenticatorTests.swift

```diff
@@ -153,6 +152,6 @@ class MediaRequestAuthenticatorTests: XCTestCase {
             XCTFail("This should not be called")
         }
 
-        waitForExpectations(timeout: 0.05)
+        waitForExpectations(timeout: 0.5)
```

Comment:
Sometimes this test takes 0.06 seconds on my Mac to finish, so I increased the timeout here a little bit. This won't affect this test's running time, since the test still finishes as soon as the expectation is fulfilled.